### ISSUE TITLE
Red screen not listed under Screen Colors

### ISSUE THREAD

**Issue by nickisnoble** [yellows8/3ds_browserhax_common#47]
I keep getting a red screen after the blue screen. No description is provided for that.

A red screen is mentioned earlier:

> At the start of this ROP-chain, the sub-screen colorfill is set to display red, at the end it's set to display blue.

But that also isn't super helpful.

**Comment by yellows8** [comment]
No, wrong section. https://github.com/yellows8/3ds_browserhax_common#screen-colors

That's from *hax payload.... The browserhax_site _specifically_ says to extract starter.zip @ SD root.
